plugins/topology/src/__fixtures__/workloadNodeData.ts

```diff
@@ -0,0 +1,3318 @@
+export const workloadNodeData = {
+  id: 'bfb6932f-64c6-4fe3-b283-16ecec8628c0',
+  type: 'workload',
+  label: 'hello-minikube2',
+  resource: {
```

Comment:
will recommend having only part of the mock data that's used in specs like width, height, and data